sql/updates/world/2015_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE `creature_template` SET `minlevel`='80', `maxlevel`='80' WHERE `entry`='30230';
+
+DELETE FROM `spell_script_names` WHERE `spell_id` IN ();
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+('61999', 'spell_dk_raise_ally_initial'),
+('46619', 'spell_dk_raise_ally');
```

Comment:
Looking at the suggestions so far, I think the SQL should look something like this in full :
```sql
UPDATE `creature_template` SET `minlevel`=80, `maxlevel`=80 WHERE `entry`=30230;
DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_dk_raise_ally_initial', 'spell_dk_raise_ally');
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
(61999, 'spell_dk_raise_ally_initial'),  
(46619, 'spell_dk_raise_ally');
```
At least this way, the queries are repeatable without causing errors upon a second run.